locales/en_short.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(number, index) {
+  return [
+    ['just now', 'a while'],
+    ['%ss ago', 'in %ss'],
+    ['1m ago', 'in 1m'],
+    ['%sm ago', 'in %sm'],
+    ['1h ago', 'in 1h'],
+    ['%sh ago', 'in %sh'],
+    ['1d ago', 'in 1d'],
+    ['%sd ago', 'in %sd'],
+    ['1w ago', 'in 1w'],
+    ['%sw ago', 'in %sw'],
+    ['1m ago', 'in 1m'],
```

Comment:
Probably it should be
```javascript
['1mo ago', 'in 1mo'],
```

Anyway I'm not sure that `1m ago` is understandable from the first view - is that a minute or a month?